app/products/calls/screens/call/call_screen.tsx

```diff
@@ -433,6 +444,21 @@ const CallScreen = (props: Props) => {
                             />
                             <Text style={style.buttonText}>{'Speaker'}</Text>
                         </Pressable>
+                        <Pressable
+                            style={style.button}
+                            onPress={toggleRaiseHand}
+                        >
+                            <HandIcon
+                                fill={props.currentParticipant?.raisedHand ? 'rgb(255, 188, 66)' : props.theme.sidebarText}
+                                height={24}
+                                width={24}
+                                style={{...style.buttonIcon, ...style.handIcon}}
```

Comment:
```suggestion
                                style={[style.buttonIcon, style.handIcon]}
```

Reply:
Is there a difference? I don't mind either way, I'll make the correction, but just wondering for my own understanding. :)